clang-tidy-checks/max-tokens-check.diff

```diff
@@ -41,24 +41,27 @@ index 000000000000..50d29634f1de
 +public:
 +  MaxTokensPPCallbacks(MaxTokensCheck *Check) : Check(Check) {}
 +
-+  void diag(SourceLocation Loc, unsigned TokenCount, unsigned MaxTokenCount) {
++  void diag(SourceLocation Loc, unsigned TokenCount, unsigned MaxTokenCount, bool SuggestFix) {
 +    if (MaxTokenCount != 0 && TokenCount > MaxTokenCount && Loc.isValid()) {
 +      std::string Msg = llvm::formatv(
 +          "Number of tokens ({0}) exceeds the specified maximum ({1})",
 +          TokenCount, MaxTokenCount);
-+      Check->diag(Loc, Msg);
++      auto BaseDiag = Check->diag(Loc, Msg);
```

Comment:
the `Loc` when using `FMaxTokens` as a CLI option is the end of the file, right? I think it makes sense that that's not fixable since it has higher precedence than the `#pragma`s